cli/cli_test.go

```diff
@@ -34,7 +34,9 @@ func TestCmdMarshalProto(t *testing.T) {
  "sticky": false,
  "group_enforced": false,
  "compressed": false,
- "cascaded": false
+ "cascaded": false,
+ "loopdev": false,
+ "ramdisk": false
```

Comment:
Are you testing that the values are always false? I think you should test for setting values to true or false, right? Who is going to take action with these values?